### PR TITLE
feat(Cassandra): Added branching doc structure

### PR DESCRIPTION
**Pull request by nbaenam**
Added the branching doc structure for this page: [Cassandra monitoring integration](https://docs.newrelic.com/docs/infrastructure/host-integrations/host-integrations-list/cassandra-monitoring-integration).

This is the ticket: NR-217458

This PR comes from this [PR: 16028](https://github.com/newrelic/docs-website/pull/16028)

### CODE REVIEW

File: src/install/cassandra/intro.mdx [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+componentType: default
+---
+
+Our Cassandra integration sends performance [metrics](#metrics) and [inventory data](#inventory) from your Cassandra database to the New Relic platform. You can view pre-built dashboards of your Cassandra metric data, create alert policies, and create your own custom queries and charts.
+
+Read on to install the integration and to see what data we collect.
+
+## Quick start [#quick]
+
+Instrument your Cassandra database quickly and send your telemetry data with guided install. Our guided install uses our infrastructure agent and our CLI to set up the Cassandra integration, and discovers other applications and log sources running in your environment and then recommends which ones you should instrument. The guided install works with most setups. But if it doesn't suit your needs, there are other install options below.
```
Comment (bradleycamacho) [inline src/install/cassandra/intro.mdx:12]: I'd put this section in a callout. Currently the intro looks a bit complicated and hard to parse, but putting this "optional" section in a callout helps break it up.

**Review by bradleycamacho — APPROVED** [review]
Looks great! Once you add the callout I mentioned, I would go ahead and merge this.

File: src/install/cassandra/intro.mdx (hunk 1 above)
Comment (nbaenam) [reply src/install/cassandra/intro.mdx]: Done!

**Comment by nbaenam** [comment]
I'm going to merge this PR because @bradleycamacho already reviewed it. I added the mentioned callout.
